copy.go

```diff
@@ -37,9 +37,9 @@ import (
 // defaultConcurrency is the default value of CopyGraphOptions.Concurrency.
 const defaultConcurrency int = 3 // This value is consistent with dockerd and containerd.
 
-// ErrSkipDesc signals to stop copying a descriptor.  When returned from PreCopy the blob must exist in the target.
+// SkipNode signals to stop copying a descriptor.  When returned from PreCopy the blob must exist in the target.
 // This can be used to signal that a blob has been made available in the target repository by "Mount()" or some other technique.
-var ErrSkipDesc = errors.New("skip descriptor")
+var SkipNode = errors.New("skip descriptor")
```

Comment:
I think we should change the text of the error to "skip node".

Reply:
changed the error text to "skip node".